readyplayerme/rpm-unreal-sdk
# include glTF content

## Description

This automate the process of this issue: https://github.com/readyplayerme/rpm-unreal-avatar-creator/issues/33#issuecomment-1710050350

I also encounter the FJsonObject include compile error in ReadyPlayerMeAmplitudeEventLogger in UE5.0.






## Changes

#### Added

-   Automatically add `/glTFRuntime` to the directory to package list.

#### Updated

-   Include `FJsonObject` definition to fix compile error in `ReadyPlayerMeAmplitudeEventLogger.h`.

#### Removed

-   Remove `FJsonObject` forward declaration in `ReadyPlayerMeAmplitudeEventLogger.h`.



## How to Test

- Remove `/glTFRuntime` from the directory to package list.
- Restart Unreal editor.
- Check the list again to see if `/glTFRuntime` exists.
- Try to package the game.



## Checklist


## QA Testing

-   Mention any useful test cases for this feature and add to QA Testing documentation.

## Details

-   If more details are necessary to support the description with images and videos add them here.
